README.md

```diff
@@ -1,43 +1,55 @@
-# Front-end Starting App
+[![Lifecycle:Experimental](https://img.shields.io/badge/Lifecycle-Experimental-339999)](https://github.com/bcgov/nr-frontend-starting-app)
 
-This is a starting front-end app that intends to get the team started with code definitions, commits standards and guides related to code-style.
+# Natural Resources Front-End Starting PWA
 
-In case you're looking for the original readme template file, you can see it [here](README_template.md)
+This repository holds a set of policies, standard, guides and pipelines to get
+started with a PWA. Before writing your first line of code, please take a
+moment and check out our [CONTRIBUTING](CONTRIBUTING.md) guide.
 
-TODO: update package.json file
+## Our Policy
 
-## Stack
+- Work in the open: That means that everything we do should be open, should be
+public. Please, don't create private repositories unless you have a very strong
+reason. Keeping things public is a must follow rule for BC Government.
+- Customer centred services: All the work that's been created is to improve all
+of our users, customers, and friends lives. Is important to keep that in mind
+because as engineers sometimes we face technical issues, however, our goal is
+to have a nice and easy daily experience.
```

Comment:
Can we rephrase this? I'm unsure of this point's intent: is it saying the work we, the people responsible for this project, do here aims to make the lives of our customers and clients, *as well as* our own, better? Who's supposed to have an easy daily experience, the customers or us?

Reply:
I'll fix it. Thanks for letting me know :)